### DISCUSSION TITLE
Feature Request: Return whether a phone number is verified

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/social-connect#223] [Feature requests (Needs)]
Surfacing this need from a recent conversation we had: 

> [It would be great to know] which of your contacts [have mapped their phone numbers]. 

I.e. given a list of phone numbers, return a boolean `True` | `False` if a phone number has a mapping ("attestation").

This might take the form of a (reverse mapping) function: `isVerifiedPhoneNumber(address _user) => bool`

This would be particularly useful to see, which contacts have verified phone numbers in an address book.

**Comment by arthurgousset** [comment]
Referencing a similar ticket for future reference: [Dapp developers should easily be able to get the verification status of an address/phone-number #7427](https://github.com/celo-org/celo-monorepo/issues/7427)

**Comment by vibern0** [comment]
This would be so amazing, really.
:pray:

**Reply by arthurgousset** [reply]
Thanks for your comment @obernardovieira! 💯 Did you upvote the post? Among other things, that helps me keep track of demand.

**Comment by arthurgousset** [comment]
When transferring this discussion topic:
- from: `celo-org/celo-monorepo/discussions`
- to: `celo-org/identity/discussions`
the upvotes were reset.

I'm including a screenshot of the upvotes for future reference: 

<img width="600" alt="image" src="https://user-images.githubusercontent.com/46296830/167673044-91cdc58a-9b0c-4275-9199-4fa319a86d2a.png">
